content/en/docs/reference/kubectl/resource_printing/raw.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Raw"
+linkTitle: "Raw"
+weight: 2
+type: docs
+description: >
+    Prints raw cluster information
+---
+
+
+{{< alert color="success" title="TL;DR" >}}
+- Get or List Raw Resources in a cluster as Yaml or Json
+{{< /alert >}}
+
+# Print Raw Resource
+
+## Motivation
+
+Inspecting or Debugging Resources.
```

Comment:
Thanks.
We've realized that a straight conversion isn't right. This proposed tweak should help a lot though:
```suggestion
```